com.dosbox.DOSBox.json

```diff
@@ -0,0 +1,66 @@
+{
+    "app-id": "com.dosbox.DOSBox",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        /* DOSBox does not respect XDG base directories */
+        "--persist=.dosbox",
+        /* Allow Documents directory access - for storing DOS apps etc. */
+        "--filesystem=xdg-documents"
+    ],
+    "command": "dosbox",
+    "rename-appdata-file": "dosbox.appdata.xml",
+    "rename-desktop-file": "dosbox.desktop",
+    "rename-icon": "dosbox",
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/man",
+        "*.la", "*.a"
+    ],
+    "modules": [
+        "shared-modules/glu/glu-9.0.0.json",
+        "shared-modules/SDL/SDL-1.2.15.json",
+        {
+            "name": "dosbox",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/project/dosbox/dosbox/0.74/dosbox-0.74.tar.gz",
+                    "sha256": "13f74916e2d4002bad1978e55727f302ff6df3d9be2f9b0e271501bd0a938e05"
+                },
+                {
+                    "type": "patch",
+                    "path": "dosbox-0.74_gcc4.6_compat.patch"
+                },
+                {
+                    "type": "file",
+                    "path": "dosbox.png"
+                },
+                {
+                    "type": "file",
+                    "path": "dosbox.appdata.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "dosbox.desktop"
+                }
+            ],
+            "post-install": [
+                "mkdir -p /app/share/appdata",
+                "cp dosbox.appdata.xml /app/share/appdata",
```

Comment:
`install -Dm644 ...` is a bit cleaner than repeated `mkdir && cp`

Reply:
Done.